airbyte-integrations/connectors/source-webflow/acceptance-test-config.yml

```diff
@@ -0,0 +1,20 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-webflow:0.1.0-alpha
```

Comment:
@sajarin take a look on this.

Reply:
@alexander-marquardt the connector image should be airbyte/source-webflow:dev instead of with versioning as you have it. Check other connectors and their acceptance-test-config.yml files for reference.

Reply:
Done